kaspresso/src/main/kotlin/com/kaspersky/kaspresso/extensions/ThrowableExt.kt

```diff
@@ -1,9 +1,9 @@
-package com.kaspersky.kaspresso.internal.extensions.other
+package com.kaspersky.kaspresso.extensions
 
 import com.kaspersky.kaspresso.internal.exceptions.KaspressoError
 import io.reactivex.exceptions.ExtCompositeException
 
-internal inline fun <reified T : Throwable> invokeSafely(
```

Comment:
Could we hide this from users somehow? Maybe with reflection or a common module?